santoku/aws/test_sqs_handler.py

```diff
@@ -86,89 +103,73 @@ def test_get_queue_url(self):
         # mocking accounts/profiles becomes stable.
         moto_aws_account = "123456789012"
 
-        expected_url = (
-            f"https://{REGION}.queue.amazonaws.com/{moto_aws_account}/{STANDARD_SQS_NAME}"
-        )
+        expected_url = f"https://{region}.queue.amazonaws.com/{moto_aws_account}/{standard_queue}"
         assert obtained_url == expected_url
 
         # Test getting the name of a queue that does not exist. Failure expected.
-        with pytest.raises(Exception) as e:
-            self.sqs_handler.get_queue_url(queue_name="WRONG_QUEUE_NAME")
+        with pytest.raises(exceptions.ClientError, match="The specified queue does not exist"):
+            sqs_handler.get_queue_url(queue_name="WRONG_QUEUE_NAME")
 
-    def test_check_message_attributes_are_well_formed(self):
+    def test_check_message_attributes_are_well_formed(self, sqs_handler, message_attributes):
         # Message attributes correctly structured. Success expected.
-        message_attributes = {
-            "TestAttribute1": {"DataType": "String", "StringValue": "Test string value",},
-            "TestAttribute2": {"DataType": "Number", "StringValue": "1000000000000000",},
-            "TestAttribute3": {"DataType": "Binary", "BinaryValue": "Test binary value",},
-        }
-
-        self.sqs_handler.check_message_attributes_are_well_formed(message_attributes)
+        sqs_handler.check_message_attributes_are_well_formed(message_attributes)
 
         # Message with more than 10 attributes. Failure expected.
         num_attributes = len(message_attributes)
         max_num_attributes = 10
-        message_content = {
+        message_attribute_content = {
```

Comment:
If this is simply the first attribute form the fixture then it should use the fixture (in case it changes in the future), i.e. `message_attributes["TestAttribute1"]`